packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -9,158 +8,156 @@ import {
 import { SystemColors } from '@microsoft/fast-web-utilities';
 import {
   accentFillRest,
+  baseHeightMultiplier,
   bodyFont,
   controlCornerRadius,
+  density,
   designUnit,
-  disabledOpacity,
+  fillColor,
   foregroundOnAccentRest,
-  neutralFillRest,
+  neutralForegroundHint,
   neutralForegroundRest,
+  strokeWidth,
   typeRampBaseFontSize,
   typeRampBaseLineHeight,
-  typeRampPlus3FontSize,
-  typeRampPlus3LineHeight
 } from '../design-tokens';
-import { heightNumber } from '../styles';
+import { DirectionalStyleSheetBehavior } from '../styles';
 
 /**
+ * LTR styles for calendar
  * @internal
  */
+const ltrStyles = css`
+.selected + .selected {
+  border-top-left-radius: 0;
+  border-bottom-left-radius: 0;
+  border-left-width: 0;
+  padding-left: calc(var(--calendar-gap) + (${strokeWidth} + ${controlCornerRadius}) * 1px);
+  margin-left: calc((${controlCornerRadius} * -1px) - var(--calendar-gap));
+}
+
+.day.disabled::before {
+  transform: translate(-50%, 0) rotate(45deg);
+}
+`;
+
+/**
+ * RTL styles for calendar
+ * @internal
+ */
+const rtlStyles = css`
+.selected + .selected {
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  border-right-width: 0;
+  padding-right: calc(var(--calendar-gap) + (${strokeWidth} + ${controlCornerRadius}) * 1px);
+  margin-right: calc((${controlCornerRadius} * -1px) - var(--calendar-gap));
+}
+
+.day.disabled::before {
+  transform: translate(50%, 0) rotate(-45deg);
+}
+`;
+
+/**
+ * Styles for calendar
+ * @public
+ */
 export const calendarStyles: (
   context: ElementDefinitionContext,
   definition: FoundationElementDefinition
 ) => ElementStyles = (
   context: ElementDefinitionContext,
   definition: FoundationElementDefinition
 ) => css`
-${display("block")} :host {
-  --cell-border: none;
-  --cell-height: calc(${heightNumber} * 1px);
-  --selected-day-background: ${neutralFillRest};
-  --selected-day-color: ${neutralForegroundRest};
-  --cell-padding: calc(${designUnit} * 1px);
-  --disabled-day-opacity: ${disabledOpacity};
-  --inactive-day-opacity: ${disabledOpacity};
+${display("inline-block")} :host {
+  --calendar-cell-size: calc((${baseHeightMultiplier} + 2 + ${density}) * ${designUnit} * 1px);
+  --calendar-gap: calc(${designUnit} * .5px);
```

Comment:
I try to discourage fractional uses of the design unit. Let's go with hard-coded `2px` for now and I'll make sure we solve this for the next density system update (coming very soon).

Reply:
It's 2px now.